packages/compartment-mapper/src/import.js

```diff
@@ -80,6 +87,7 @@ export const loadLocation = async (readPowers, moduleLocation, options) => {
       parserForLanguage,
       globals,
       modules,
+      attenuations,
```

Comment:
Should we call them attenuations or attenuators? My initial choice may have been wrong.